app/src/main/java/com/example/routinely/ui/components/LoginTextField.kt

```diff
@@ -1,34 +1,50 @@
 package com.example.routinely.ui.components
 
 import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.material3.MaterialTheme
 import androidx.compose.material3.OutlinedTextField
 import androidx.compose.material3.OutlinedTextFieldDefaults
 import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
 import androidx.compose.runtime.getValue
 import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
 import androidx.compose.runtime.saveable.rememberSaveable
 import androidx.compose.runtime.setValue
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.input.TextFieldValue
 import androidx.compose.ui.tooling.preview.Preview
+import com.example.routinely.login.isValidEmailFormat
 import com.example.routinely.ui.theme.Gray80
 
 @Composable
 fun LoginTextField(onEmailChange: (String) -> Unit) {
-    var text by rememberSaveable { mutableStateOf("") }
+    var text by remember { mutableStateOf(TextFieldValue()) }
```

Comment:
Just check if it is necessary

Reply:
ok, this has been removed, there really was no need